src/PresentationalComponents/SystemsTable/SystemsTable.js

```diff
@@ -285,7 +285,8 @@ const SystemsTable = () => {
             filters,
             selectedTags,
             workloads,
-            SID
+            SID,
+            true
```

Comment:
Can we refactor this function so that it does not use a constant boolean for detecting the systems page? I am having difficulties on how to write it down here, feel free to ping me if more explanation is needed.